docs/docsite/rst/dev_guide/developing_module_utilities.rst

```diff
@@ -4,75 +4,142 @@
 Appendix: Module Utilities
 **************************
 
-Ansible provides a number of module utilities that provide helper functions that you can use when developing your own modules. The ``basic.py`` module utility provides the main entry point for accessing the Ansible library, and all Python Ansible modules must, at minimum, import ``AnsibleModule``::
+A mechanism exists to ship utility modules imported by a :ref:`New-Style
+Module <flow_new_style_modules>` alongside for execution on the target. When a
+new-style module executes, utility modules may be imported via the
+``ansible.module_utils`` Python package namespace. The module entry point is
+imported from this namespace, and must always be present:
```

Comment:
Note that always be present is no longer true.  You must import something from module_utils but it doesn't matter what.